modules/admin-ui/src/main/java/org/opencastproject/adminui/endpoint/AbstractEventEndpoint.java

```diff
@@ -1513,7 +1513,7 @@ public Response getEventWorkflows(@PathParam("eventId") String id)
       return notFound("Cannot find an event with id '%s'.", id);
 
     try {
-      if (optEvent.get().isScheduledEvent() && !optEvent.get().hasRecordingStarted()) {
+      if (optEvent.get().getEventStatus().contains("SCHEDULE")) {
```

Comment:
I think what you really want here is...

`if (optEvent.get().getEventStatus().equals("EVENTS.EVENTS.STATUS.SCHEDULED")) {`

... although I am somewhat shocked to see a string constant being used in https://github.com/opencast/opencast/blob/develop/modules/index-service/src/main/java/org/opencastproject/index/service/impl/index/event/Event.java#L882.

... but that is another issue.